src/components/Contacts/ContactDetails/ContactDetailsHeader/DeleteContactModal/DeleteContactModal.tsx

```diff
@@ -187,6 +188,21 @@ export const DeleteContactModal: React.FC<DeleteContactModalProps> = ({
                 </ListItem>
               )}
             </List>
+            <br />
+            <Typography>
+              <Link
+                href={`mailto:${
+                  process.env.DONATION_SERVICES_EMAIL
+                }?subject=Request+contact+deletion&body=${encodeURIComponent(
+                  `Dear Donation Services,\n\Please could you remove the following contact: ${contactName} ` +
+                    '\n\nThanks,\n\n',
+                )}`}
+                underline="hover"
+                sx={{ fontWeight: 'bold' }}
+              >
+                {t('Email Donation Services here')}
```

Comment:
Instead of having a separate link at the bottom, would it be possible to make the `email Donation Services` phrase in `For contacts originating from Donation Services or DonorHub, email Donation Services to request deletion.` the email link?

Reply:
Good call